app/controllers/buyers/applications/bulk/change_states_controller.rb

```diff
@@ -22,4 +23,10 @@ def create
     handle_errors
   end
 
+  private
+
+  def change_state_action_param
+    params.require(:change_states).require(:action)
```

Comment:
We need to make sure if we do not select any action that `params.require(:change_states) == {}`